tests/test_kinesisvideo/test_kinesisvideo.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import unicode_literals
+
+import boto3
+import sure  # noqa
+from nose.tools import assert_raises
+from moto import mock_kinesisvideo
+from botocore.exceptions import ClientError
+import json
+
+
+@mock_kinesisvideo
+def test_list():
```

Comment:
Can we split this test up? I prefer to have smaller test cases, i.e. `test_list`, `test_create`, `test_create_with_same_name`, etc.

Reply:
done: https://github.com/spulec/moto/pull/3271/commits/937dd23ec984cab27ec154dffc93d7c6bf0485ba